signalfx/util.go

```diff
@@ -50,6 +50,22 @@ var ChartColorsSlice = []chartColor{
 	{"lime_green", "#6bd37e"},
 }
 
+// getPayload for viz objects is called withs: *schema.ResourceDiff or *schema.ResourceData - that's why we need this interface
+type ResourceDataAccess interface {
+	Get(key string) interface{}
+	GetOk(key string) (interface{}, bool)
+}
+
```

Comment:
I would prefer to not add an interface within the utility file here, it will make it hard to maintain since it is somewhat hidden within the file of all the things that don't exactly fit within a resource object.

To be clear, the interface is fine but move it to a seperate file to make it easier to maintain and discover later on. 

Moreover, Go doc style should take the form of:

```golang
// <Definition> ...
<Definition>
```

Please update to be reflect the go doc style.